perma_web/frontend/components/CreateLink.vue

```diff
@@ -0,0 +1,114 @@
+<script setup>
+import { ref, watch } from 'vue'
+import { globalStore } from '../stores/globalStore'
+import { getCookie } from '../../static/js/helpers/general.helpers'
+// import { Spinner } from 'spin.js'
+// Cannot import spinner like this just yet
+
+const userLink = ref('')
+const userLinkGUID = ref('')
+const userLinkStepCount = ref(0)
+// const spinner = new Spinner({ lines: 15, length: 2, width: 2, radius: 9, corners: 0, color: '#2D76EE', trail: 50, top: '12px' });
```

Comment:
The spinner code here is temporarily commented out — 

`import { Spinner } from 'spin.js` 

Is the recommended way to import spin.js, according to its documentation, but attempting to use `new Spinner` returns a build error that `Spinner` is not a constructor. It may be easier to simply adopt a slightly more modern spinner, although implementing any kind of animation that persists for longer than 5 seconds without a play/pause button is generally understood to be an accessibility failure under the requirements of [WCAG 2.2.2 success criteria](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html).

Reply:
I haven't dug further into this, but will!

Reply:
Looks like we can easily fix this by diverging from the docs and just importing spinner as: 

```
import * as spinner from 'spin.js'
```